src/render_gl1.c

```diff
@@ -174,13 +178,16 @@ static void gl1_resize_screen(struct graphics_data *graphics,
   gl1.glBindTexture(GL_TEXTURE_2D, render_data->texture_number);
   gl_check_error();
 
+  gl1.glTexImage2D(GL_TEXTURE_2D, 0, GL_RGBA, GL_POWER_2_WIDTH,
+   GL_POWER_2_HEIGHT, 0, GL_RGBA, GL_UNSIGNED_BYTE, NULL);
+
   gl_set_filter_method(graphics->gl_filter_method, gl1.glTexParameterf);
 }
 
 static boolean gl1_set_video_mode(struct graphics_data *graphics,
  int width, int height, int depth, boolean fullscreen, boolean resize)
 {
-  int internal_width = GL_POWER_2_WIDTH, internal_height = GL_POWER_2_HEIGHT;
+  int internal_width = CHAR_W * SCREEN_W, internal_height = CHAR_H * SCREEN_H;
```

Comment:
Would prefer `SCREEN_PIX_W` and `SCREEN_PIX_H` here. Looks good otherwise.

Reply:
Fixed.